libclc/CMakeLists.txt

```diff
@@ -337,6 +348,10 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
         list( APPEND flags -Xclang -target-feature -Xclang +avx
             -Xclang -fsycl-is-native-cpu
             -Xclang -target-feature -Xclang +avx512f)
+      elseif( ARCH  STREQUAL aarch64)
+        list( APPEND flags -Xclang -fsycl-is-native-cpu)
+      else()
```

Comment:
Do we need this `else()`? We already have `set (flags )` above.

Reply:
Messed up when rebasing, thanks for spotting it